code/game/atoms_movable.dm

```diff
@@ -754,4 +754,92 @@
 	appearance = falltempoverlays[src]
 	falltempoverlays -= src
 	ignoreinvert = initial(ignoreinvert)
-	invisibility = init_invisibility
\ No newline at end of file
+	invisibility = init_invisibility
+
+/datum/proc/send_to_past(var/duration)
+	return
+
+/datum/var/being_sent_to_past = FALSE
```

Comment:
Of course, but I'm iffy on making this be a variable on the most low level of types because it does incur a performance overhead on everything.